package/cluster/definition.yaml

```diff
@@ -30,6 +30,37 @@ spec:
                 type: object
                 description: Cluster configuration parameters.
                 properties:
+                  region:
+                    type: string
+                    default: us-west-2
+                    description: Amazon AWS region
```

Comment:
Please mention the cloud provider documentation URL in the description so that it will be clear where the region list is coming from.